mkdocs_rss_plugin/config.py

```diff
@@ -44,6 +44,7 @@ class RssPluginConfig(Config):
     )
     comments_path = config_options.Optional(config_options.Type(str))
     date_from_meta = config_options.SubConfig(_DateFromMeta)
+    description = config_options.Type(str, default="")
```

Comment:
It would be better to use None as default value:

```suggestion
    description =  config_options.Optional(config_options.Type(str))
```